
float32 min_angle
float32 max_angle
bool external_angle_sensing

---

string[] cloud_names
sensor_msgs/PointCloud2[] output_clouds